alvr/server_openvr/src/lib.rs

```diff
@@ -149,19 +140,73 @@ extern "C" fn driver_ready_idle(set_default_chap: bool) {
                     }) = controllers_config
                     {
                         (
-                            hand_skeleton_config.use_separate_trackers,
+                            hand_skeleton_config.steamvr_input_2_0,
                             left_openvr_hand_skeleton.map(tracking::to_ffi_skeleton),
                             right_openvr_hand_skeleton.map(tracking::to_ffi_skeleton),
                         )
                     } else {
                         (false, None, None)
                     };
 
-                    let ffi_motions = tracking
+                    let use_left_hand_tracker = use_separate_hand_trackers
+                        && tracking.hand_skeletons[0].is_some()
+                        && tracking
+                            .device_motions
+                            .iter()
+                            .all(|(id, _)| *id != *HAND_LEFT_ID);
```

Comment:
If this works how I think it works then it's a really bad way to check for multimodal input

Reply:
If the api doesn't allow it without too much rewriting then fine, but it should be documented (given that I'm actually right ofc)

Reply:
So, to clarify, server_openvr always read input with ultimodal protocol. in here, we are checking that only hands are active && we want separate trackers. if we don't want separate trackers, or we have only controllers, or we have both controllers and hand tracking, use steamvr controller devices.

Reply:
right, we don't want 2.0 input if we're getting both because that would be multimodal, which clashes
could maybe use a comment, but should be fine ig

Reply:
Well technically there is no clash between SteamVR input 2.0 and multimodal. SteamVR naturally supports it

Reply:
Yea but we don't want it to happen, right?

Reply:
There is not reason to avoid it. Steamvr 2.0 + multimodal will use hand trackers when using free hands, and controllers with hand tracking (as estimated skeletal level) for controllers

Reply:
Nvm I get what's going on. My mistake was forgetting that detached controllers get send with detached controllers ids from the client already (right? (I'm losing track help)), I thought the redistribution was happening on the server.

Reply:
Detached controllers are actually not sent yet, because they are useless at the moment. We only send when controllers are in hand + hand tracking data

Reply:
right, but it functionally amounts to the same thing, all good then